src/Symfony/Component/Form/Tests/Extension/Core/Type/ChoiceTypeTest.php

```diff
@@ -490,6 +490,21 @@ public function testSubmitSingleNonExpandedObjectChoices()
     /**
      * @group legacy
      */
+    public function testLegacyNullChoices()
+    {
+        $form = $this->factory->create('choice', null, array(
+            'multiple' => false,
+            'expanded' => false,
+            'choices' => null,
+        ));
+        $this->assertEquals($form->getConfig()->getOption('choices'), null);
```

Comment:
I'm not sure but I think `assertEquals` doesn't make a strict comparison, so assertion will be valid you have `false` and expect `null` (and vice-versa). I suggest to use the proper assertion methods for both readability and strict comparison.

```
$this->assertNull($form->getConfig()->getOption('choices'));
$this->assertFalse($form->getConfig()->getOption('multiple'));
$this->assertFalse($form->getConfig()->getOption('expanded'));
```